src/api/spec/models/token/workflow_spec.rb

```diff
@@ -121,7 +121,7 @@
       end
 
       it 'raises an "Invalid workflow step definition" error' do
-        expect { subject }.to raise_error('Invalid workflow step definition')
+        expect { subject }.to raise_error('Invalid workflow step definition: ')
```

Comment:
This is kind of weird due to the extra colon, but it happens only since the steps are empty. I would say it's fine.

Reply:
We can improve later :+1: